scimma_admin/hopskotch_auth/auth.py

```diff
@@ -13,6 +14,12 @@ class NotInKafkaUsers(PermissionDenied):
 
 class HopskotchOIDCAuthenticationBackend(auth.OIDCAuthenticationBackend):
     """Subclass Mozilla's OIDC Auth backend for custom hopskotch behavior. """
+    def __init__(self):
+        auth.OIDCAuthenticationBackend.__init__(self)
+        if "KAFKA_USER_AUTH_GROUP" in os.environ:
+            self.kafka_user_auth_group=os.environ["KAFKA_USER_AUTH_GROUP"]
+        else:
+            self.kafka_user_auth_group="kafkaUsers"
```

Comment:
Please move this logic to `scimma_admin/scimma_admin/settings.py`. Then, you can reference the set value through `django.conf.settings`. For example:

```python
# file: scimma_admin/scimma_admin/settings.py

KAFKA_USER_AUTH_GROUP = os.environ.get("KAFKA_USER_AUTH_GROUP", default="kafkaUsers")
```
```python
# file: scimma_admin/hopskotch_auth/auth.py
from django.conf import settings

class HopskotchOIDCAuthenticationBackend(auth.OIDCAuthenticationBackend):
    """Subclass Mozilla's OIDC Auth backend for custom hopskotch behavior. """
    def __init__(self):
        auth.OIDCAuthenticationBackend.__init__(self)
        self.kafka_user_auth_group = settings.KAFKA_USER_AUTH_GROUP
```

I prefer this because putting all configurable options in one settings file makes it easier to see which values could be different when trying to debug differences in behavior in different environments.

Reply:
Sidenote - if it's in `settings.py`, you might not want to use `os.environ`. You might just "hardcode" it, and let it get overwritten with a `local_settings.py` file (as expressed [here](https://github.com/scimma/scimma-admin/blob/777024ffc4bf68b7dd17c45177943ff8a4176817/scimma_admin/scimma_admin/settings.py#L219)). This would be more typical of Django code, but it doesn't make a big difference to me.

Reply:
Done; keeping the settings in one place is a good point. 

I think that unless you feel strongly we want the option to read it from the environment, as this is much easier to configure from Kubernetes. We could also do the local_settings method, but that would involve having to inject the file as a configmap.